argo-messaging.spec

```diff
@@ -0,0 +1,63 @@
+#debuginfo not supported with Go
+%global debug_package %{nil}
+
+Name: argo-messaging
+Summary: ARGO Messaging API for broker network
+Version: 0.9.0
+Release: 1%{?dist}
+License: ASL 2.0
+Buildroot: %{_tmppath}/%{name}-buildroot
+Group:     EGI/SA4
+Source0: %{name}-%{version}.tar.gz
+BuildRequires: golang
+BuildRequires: bzr
+BuildRequires: git
+ExcludeArch: i386
+Obsoletes: argo-messaging
+
+%description
+Installs the ARGO Messaging API
+
+%prep
+%setup
+
+%build
+export GOPATH=$PWD
+cd src/github.com/ARGOeu/argo-messaging/
+go get github.com/tools/godep
+godep restore
+godep update ...
+go install
+
+%install
+%{__rm} -rf %{buildroot}
+install --directory %{buildroot}/var/www/argo-web-api
+install --mode 755 bin/argo-messaging %{buildroot}/var/www/argo-messaging/argo-messaging
+
+install --directory %{buildroot}/etc
+install --mode 644 src/github.com/ARGOeu/argo-messaging/config.json %{buildroot}/etc/argo-messaging/config.json
+
+install --directory %{buildroot}/etc/init
+install --mode 644 src/github.com/ARGOeu/argo-messaging/argo-messaging.conf %{buildroot}/etc/init/
```

Comment:
This file (`argo-messaging.conf`) is not part of this PR. Better add an init script configuration file as part of this PR.